### PR TITLE
Update irsa chapter

### PR DESCRIPTION
**Pull request by davivcgarcia**
*Issue #, if available:*

#1211

*Description of changes:*

This PR is part of the internal efforts to update the workshop to the latest versions of EKS and tooling.

### CODE REVIEW

**Comment by trevorrobertsjr** [comment]
Reviewing...

**Review by trevorrobertsjr — COMMENTED** [review]
@davivcgarcia please update the following items in /110_irsa/preparation.md, and then I'll approve:

1. Kubectl Client Version number will be v1.20.4-eks-6b7464 based on the updated workshop prerequisites instructions. In the info box, maybe say something similar "your output may vary, but you need to be more recent than 1.16"
2. With the EKS Workshop prereq steps, the attendee's cli version will be `aws-cli/1.19.112 Python/2.7.18 Linux/4.14.232-176.381.amzn2.x86_64 botocore/1.20.112` based on the output from my installation on Cloud9. If you haven't done so for this lab, please verify CLI outputs using a Cloud9 environment.

**Comment by davivcgarcia** [comment]
> @davivcgarcia please update the following items in /110_irsa/preparation.md, and then I'll approve:
> 
>     1. Kubectl Client Version number will be v1.20.4-eks-6b7464 based on the updated workshop prerequisites instructions. In the info box, maybe say something similar "your output may vary, but you need to be more recent than 1.16"
> 
>     2. With the EKS Workshop prereq steps, the attendee's cli version will be `aws-cli/1.19.112 Python/2.7.18 Linux/4.14.232-176.381.amzn2.x86_64 botocore/1.20.112` based on the output from my installation on Cloud9. If you haven't done so for this lab, please verify CLI outputs using a Cloud9 environment.

Done! Please let me know if we are good to merge.

File: content/beginner/110_irsa/preparation.md [hunk 1]
```diff
@@ -1,44 +1,50 @@
...
 
+Output:
+{{< output >}}
+Client Version: v1.20.4-eks-6b7464
+Server Version: v1.20.4-eks-6b7464
+{{< /output >}}
+
 {{% notice info %}}
-If your aws cli version is lower than 1.18.15, use [Installing the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) in the User Guide
+If your aws cli version is lower than 1.20.3, use [Installing the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) in the User Guide
```
Comment (trevorrobertsjr) [inline content/beginner/110_irsa/preparation.md:27]: @davivcgarcia this item needs to be updated to 1.19.122 as well.

**Review by trevorrobertsjr — COMMENTED** [review]
One AWS CLI version reference was missed, @davivcgarcia. I will check back later today to approve once you have updated. Thanks!

File: content/beginner/110_irsa/preparation.md (hunk 1 above)
Comment (davivcgarcia) [reply content/beginner/110_irsa/preparation.md]: Done!

**Review by buzzsurfr — APPROVED** [review]
LGTM :shipit:

**Pull request merged**